x-pack/plugins/security_solution/public/resolver/store/middleware.ts

```diff
@@ -92,6 +93,27 @@ export const resolverMiddlewareFactory: MiddlewareFactory = (context) => {
           });
         }
       }
+    } else if (action.type === 'userRequestedRelatedEventData' && context) {
```

Comment:
note: this will have a different action name soon. right?

Reply:
https://github.com/elastic/kibana/pull/69098/commits/1058e680726ef3f49b925e610a67cf750a43c827